Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -1880,4 +1907,27 @@ + (NSString *)kitDisplayVersion {
 	return BNC_SDK_VERSION;
 }
 
+#pragma mark - Crashlytics reporting enhancements
+
++ (void)enhanceCrashlyticsReports
+{
+    /* Not consistently showing up. Moving to a later method.
+    [self addBranchSDKVersionToCrashlyticsReport];
```

Comment:
Removed. The placement of these calls is debatable. The idea is to get them in as early as possible. I imagine this will evolve.